cdc/sink/table_sink.go

```diff
@@ -116,15 +137,15 @@ func (t *tableSink) EmitCheckpointTs(_ context.Context, _ uint64, _ []model.Tabl
 
 // Init table sink resources
 func (t *tableSink) Init(tableID model.TableID) error {
-	return t.manager.bufSink.Init(tableID)
+	return t.backendSink.Init(tableID)
 }
 
 // Close once the method is called, no more events can be written to this table sink
 func (t *tableSink) Close(ctx context.Context) error {
-	return t.manager.destroyTableSink(ctx, t.tableID)
+	return t.Barrier(ctx, t.tableID)
 }
 
 // Barrier is not used in table sink
 func (t *tableSink) Barrier(ctx context.Context, tableID model.TableID) error {
-	return nil
+	return t.backendSink.Barrier(ctx, tableID)
```

Comment:
Will the backend sink guarantee that no more data whose `commit-ts` is larger than the max resolved ts(which is decided by now) will be written to downstream after `Barrier` is called.

Reply:
We paused the data to continue sending before we called barrier, so there should be no new data after we're done.
@sdojjy What do you think?

Reply:
For pipeline: https://github.com/pingcap/tiflow/blob/033e7d49c7f21be5722009f12dfe3b5b991b2c1a/cdc/processor/pipeline/sink.go#L288


For actor: https://github.com/pingcap/tiflow/pull/5248/files